data/tekton-pipelines/okd/windows-customize.yaml

```diff
@@ -126,23 +126,25 @@ data:
 apiVersion: tekton.dev/v1beta1
 kind: Pipeline
 metadata:
-  name: windows10-customize
+  name: windows-customize
+  annotations:
+    description: "windows-customize pipeline provides functionality for windows VMs to e.g. install sql server, or install VS Code into windows VM. This pipeline can be reused for customizing Windows 10 and 11."
 spec:
   params:
     - name: sourceTemplateName
-      description: Name of the Template which is used to customize Windows 10 and create a golden Template. The default boot source of this template needs to be available.
+      description: Name of the Template which is used to customize Windows and create a golden Template. The default boot source of this template needs to be available.
       type: string
       default: windows10-desktop-large
     - name: sourceTemplateNamespace
-      description: Namespace of the Template which is used to customize Windows 10 and create a golden Template. The default boot source of this template needs to be available.
+      description: Namespace of the Template which is used to customize Windows and create a golden Template. The default boot source of this template needs to be available.
       type: string
       default: openshift
     - name: customizeConfigMapName
-      description: Name of the ConfigMap containing the customization and sysprep configuration files (unattend.xml, etc.).
+      description: "Name of the ConfigMap containing the customization and sysprep configuration files (unattend.xml, etc.). Possible values are: windows10-sqlserver or windows11-vs-code"
```

Comment:
I would change it to "For example windows10-sqlserver or windows11-vs-code". It is possible to provide customize ConfigMaps created by the user too.